src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -226,6 +237,9 @@ public function configureOptions(OptionsResolver $resolver)
             // this option.
             'data_class' => null,
             'compound' => $compound,
+            'empty_data' => function (Options $options) {
+                return $options['compound'] ? array() : '';
```

Comment:
Hi @nicolas-grekas ,
Could you explain, why empty data is empty string instead of null as it was before?

Reply:
We are using [DateTimeToRfc3339Transformer](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToRfc3339Transformer.php#L32-L38), and it's expecting null as an empty value, otherwise throws an exception, and probably the same issue will be with [DateTimeToLocalizedStringTransformer](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php#L80-L86)

Reply:
`empty_data` is in the view format. This value should not be passed to `transform()` but to `reverseTransform()`. Can you please open a new issue with fine steps that allow to reproduce if you think you found a bug?

Reply:
Hm, based on examples from the [documentation](https://symfony.com/doc/current/reference/forms/types/form.html#empty-data) I thought `empty_data` is a model format, not view, as it could be  `new $data_class()`. 

Need to investigate then. 
Thank you for the fast response!